cli/cli_app.go

```diff
@@ -233,7 +233,7 @@ func downloadModules(terragruntOptions *options.TerragruntOptions) error {
 			return err
 		}
 		if shouldDownload {
-			return shell.RunTerraformCommand(terragruntOptions, "get", "-update")
+			return shell.RunTerraformCommandAndIgnoreOutput(terragruntOptions, "get", "-update")
```

Comment:
Implementing a different logging mechanism is beyond this PR. I will make changes to always keep the ouput, but isolate the significant output from the artefacts.